src/components/styles/StyledLink.tsx

```diff
@@ -1,32 +1,16 @@
 import { css } from "@emotion/react";
-import styled from "@emotion/styled";
-import { palette } from "@leafygreen-ui/palette";
+import { Link as LGLink } from "@leafygreen-ui/typography";
 import { Link } from "react-router-dom";
-import { fontSize } from "constants/tokens";
 
-const { blue } = palette;
+export const StyledLink = (props) => (
+  <LGLink
+    hideExternalIcon
+    css={css`
+      // Override LG's fixed 13px line height
+      line-height: inherit;
+    `}
+    {...props}
+  />
+);
```

Comment:
This might be a mistake; the color of the tab turns blue when I start clicking on them :0

<img width="769" alt="Screen Shot 2022-10-26 at 8 24 07 PM" src="https://user-images.githubusercontent.com/47064971/198162903-697b1867-4d48-4d51-bd02-fb99a0cf7914.png">